config_sample/config.json

```diff
@@ -26,11 +37,12 @@
                 1
             ],
             "spid_user_attributes": [
-                "name",
-                "familyName",
-                "email",
-                "fiscalNumber"
+                "given_name",
+                "family_name",
+                "birthdate",
+                "https://attributes.eid.gov.it/fiscal_number"
             ],
+            "aud": "https://preproduzione.oidc.idserver.servizicie.interno.gov.it https://preproduzione.oidc.idserver.servizicie.interno.gov.it/idp/profile/oidc/authorize",            

```

Comment:
necessario qui?

Reply:
no, prendo $op_id e $authorization_endpoint e li metto in array